server/chat/backend/agent/tools/github_rca_tool.py

```
@@ -202,6 +210,60 @@ def _parse_mcp_response(result: Dict[str, Any]) -> Dict[str, Any]:
     return result
 
 
+def _call_github_api(
```

Ruff (0.15.12)

[warning] Too many return statements (8 > 6)

(PLR0911)